lib/App/DuckPAN/Web.pm

```diff
@@ -539,6 +541,19 @@ sub request {
 	return $response;
 }
 
+
+sub run_on_ready_script {
+	my ($str) = @_;
+
+	# any unescaped script tags at this stage will XSS the page - so strip them
+	$str =~ s|</?script>||gi;
```

Comment:
@moollaza I get very nervous w/ Regex/HTML; it's just asking for trouble...

Something like `Mojo` should catch all the weird edge-cases w/o us having to worry any more; I stand by my statement.